buildenv/jenkins/JenkinsfileBase

```diff
@@ -1291,6 +1291,8 @@ def getGitRepoBranch(ownerBranch, defaultOwnerBranch, repo) {
 		String[] tokens = ownerBranch.split(":")
 		if (tokens.size() == 2 ) {
 			actualRepoBranch = tokens;
+		} else {
+			error "[ERROR]: Wrong format of Owner and Branch. Please use the format as [owner]:[branch]"
```

Comment:
Since we print out the error message, please also print the `ownerBranch` value. For example:
```
error "[ERROR]: Wrong format of XXX_OWNER_BRANCH value: ${ownerBranch}. The expected format is [owner]:[branch]"
```